src/modules/expression-evaluator/package.json

```diff
@@ -30,7 +30,8 @@
   },
   "scripts": {
     "build": "rimraf ./lib && tsc",
-    "test": "jest"
+    "test": "jest",
```

Comment:
Does this really need a test environment?

Reply:
Oh! Thaat's the copy of the expression-evaluator. Got it!

Reply:
I'm still not convinced that having a copy instead of using the package is better.

Reply:
> I'm still not convinced that having a copy instead of using the package is better.

Because if we're doing development on the module itself and want to test queries, we the GUI to display using the in-development version so we can see the effect, not just what would be shown with the current published package.